bindata/network/ovn-kubernetes/ipsec.yaml

```diff
@@ -238,10 +238,20 @@ spec:
             - -c
             - |
               #!/bin/bash
-              ovs-appctl -t ovs-monitor-ipsec ipsec/status
+              ovs-appctl -t ovs-monitor-ipsec ipsec/status && ipsec status
           initialDelaySeconds: 120
           periodSeconds: 30
           timeoutSeconds: 40
+        livenessProbe:
```

Comment:
So readiness and liveliness probe are the same for the ipsec containers?

Reply:
Yes.

Reply:
Is that alright? I'm not super familiar but I assume we would rather have two checks?

Reply:
A readiness probe should indicate when a pod is ready to receive traffic. This makes sense when you are building a service on top of the cluster as it will indicate when the service is ready to accept traffic. I am not sure how much it is a requirement for these networking pods. I don't think it hurts as it will at least give an indication when the pod is ready to receive IKE traffic (i.e. when the IPsec daemons are up). The liveness probe is for detecting failures of the pod. At the moment, I think the biggest failure that I want to detect is the daemons just falling over (e.g. with a segfault) so I think this liveness probe is appropriate. They are the same now but maybe in the future they might change. I don't think that I see an issue with that.